common_dwp_api/src/main/java/uk/gov/dhsc/htbhf/dwp/model/QualifyingBenefits.java

```diff
@@ -11,6 +11,7 @@
     JOBSEEKERS_ALLOWANCE,
     PENSION_CREDIT,
     CHILD_TAX_CREDIT,
+    UNDER_18,
```

Comment:
I get why this is here, but it's a bit smelly having this in QualifyingBenefits, is there nowhere else this could go?

Reply:
This is providing the ability to override the reason why the applicant qualifies. It would be better if `QualifyingBenefits` were called `QualifyingReason` in this circumstance, but it's providing the same function. It's where Karen suggested it go, and gets reported to Google Analytics. If we put it somewhere else (a new field, say) then we'd have to modify the reporting code too.